src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -321,6 +323,24 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
         chart.on("data", handleLayerLoaded);
     };
 
+    private adjustChartHeight = () => {
+        const { chart, chartRef } = this;
+        if (!chartRef) {
+            return;
+        }
+
+        const chartHeight: number = chartRef.clientHeight;
+        const parentHeight: number = chartRef.parentElement.clientHeight;
+        const shouldResize: boolean = chartHeight <= MIN_HEIGHT && MIN_HEIGHT <= parentHeight;
```

Comment:
Where the MIN_HEIGHT come from?

Reply:
It includes zoom control height (60xp) and its margins (20px for top and bottom respectively)
More [detail](https://github.com/gooddata/gooddata-react-components/pull/1596/files#diff-fa62e9a6886bb2448106a41bbcfa6e89R152)